projects/commudle-admin/src/app/components/home/communities/home-community-card/home-community-card.component.scss

```diff
@@ -48,3 +48,55 @@
     }
   }
 }
+
+
+@media only screen and (max-width: 768px){
+  nb-card {
+    width: 100%;
+    height: 120px !important;
+  }
+   nb-card-body {
+      display: flex;
+      flex-direction: row !important;
+      justify-content: space-between;
+      padding-top: 0;
+      padding-bottom: 0;
+    }
+      .left-column {
+        flex: 30%;
+       
+      }
+      .right-column{
+        // white-space: nowrap;
+        // text-align : left;
+        // text-overflow: ellipsis;
+        // flex: revert;
+      }
+      .logo {
+        display: inline-flex;
+        align-items: center;
+        img {
+          margin-right: 10px;
+        }
+      }
+
+      .name {
+        text-align: left !important;
+        font-size: 18px;
```

Comment:
Let's get the font sizes from variables.scss, if this size is not available, then create a variable